day_two_guide/topics/increasing_docker_storage.adoc

```diff
@@ -127,6 +141,43 @@ ose-app-node01.example.com   Ready                      24m       v1.6.1+5115d70
 ose-app-node02.example.com   Ready                      24m       v1.6.1+5115d708d7
 ----
 
+*Increasing storage with a new disk*
+
+. Evacuate the node xref:evacuating-a-node[following the steps above].
+
+. Stop the `docker` and `atomic-openshift-node` services:
++
+----
+# systemctl stop docker atomic-openshift-node
+----
+
+. Resize the disk.
```

Comment:
I'd say:

Depending on your environment, the process can be different. If running in a cloud provider, it would be easy to dettach the disk, destroy the disk, create a new bigger disk and attach it to the instance. Otherwise, the disk can be resized as well as the filesystem. See https://access.redhat.com/solutions/199573 for more information.